kubectl/pod.go

```diff
@@ -74,7 +74,7 @@ type ContainerStatus struct {
 	Image                string         `yaml:"image"`
 	ImageID              string         `yaml:"imageID"`
 	ContainerID          string         `yaml:"containerID,omitempty"`
-	Started              *bool          `yaml:"started,omitempty"`
```

Comment:
Sounds really weird to have a pointer here, as is the same kind of variable as `Ready`. As far as I can see it's not used anywhere yet, so I think it's safe to modify it.

Reply:
Usually the pointer is used to distinguish three different states: true, false and undefined (that would be a null value). Can be useful to prevent a default `false` value, as boolean is initiated to false by default in golang. So that if mapping a resource with undefined Started status in k8s this will be mapped always to a `false` value. I don't think this is a problem in this case.

Reply:
OK, thanks for the explanation!

So in that case maybe I can keep it as `*bool` and use this test in `kubectl.go`:
```go
if !*s.Started {
	return false, nil
}
```

It should work, no?

Reply:
So, as everybody agreed, I will let it as is.